breathe/renderer/sphinxrenderer.py

```diff
@@ -287,16 +366,157 @@ def get_filename(node):
             filename = get_filename(file_data.compounddef)
         return self.project_info.domain_for_file(filename) if filename else ''
 
+    def join_nested_name(self, names):
+        dom = self.get_domain()
+        sep = '::' if not dom or dom == 'cpp' else '.'
+        return sep.join(names)
+
+    def run_directive(self, obj_type, names, contentCallback, options={}):
+        args = [obj_type, names] + self.context.directive_args[2:]
+        directive = DomainDirectiveFactory.create(self.context.domain, args)
+        assert issubclass(type(directive), BaseObject)  #
```

Comment:
Odd comment start mark `#` at the end, should there be one or was it removed during development?

Reply:
Odd indeed, removed.